internal/scripts/scripts_test.go

```diff
@@ -77,6 +88,31 @@ var scriptsTests = []scriptsTest{{
 		"/bar/":          "dir 0755",
 		"/bar/file3.txt": "file 0644 5b41362b",
 	},
+	mutated: []string{
+		"/foo/file1.txt",
+		"/foo/file2.txt",
+	},
+}, {
+	summary: "Check mode is retained",
+	content: map[string]string{
+		"foo/file1.txt": ``,
+		"foo/file2.txt": ``,
+	},
+	hackdir: func(c *C, dir string) {
+		fpath1 := filepath.Join(dir, "foo/file1.txt")
+		os.Chmod(fpath1, 0744)
+	},
+	script: `
+		content.write("/foo/file1.txt", "data")
```

Comment:
For the test to assert that the mode is preserved we need to content.write to file2 as well.

Reply:
Done: https://github.com/letFunny/chisel/pull/8/commits/7ce244dbeb861b65ed99d1c4e64ceb4787e11d7e.